qiita_db/util.py

```diff
@@ -612,6 +612,28 @@ def get_mountpoint(mount_type, conn_handler=None, retrieve_all=False):
     return [(d, join(basedir, m, s)) for d, m, s in result]
 
 
+def get_mountpoint_path(mount_id, conn_handler=None):
```

Comment:
This can be confused with get_mountpoint, perhaps change to get_mountpoint_path_by_id?